articles/quickstart/native/ios-swift/07-linking-accounts.md

```diff
@@ -83,9 +53,9 @@ Auth0
     }
 ```
 
-## Retrieve Linked Accounts
+## Retrieve the Linked Accounts
 
-Linked accounts, a.k.a. the user's identities, can be retrieved by fetching the user's profile, a process that we already know from the [User Sessions Tutorial](/quickstart/native/ios-swift/03-user-sessions#validate-an-accesstoken):
+You can retrieve the linked accounts (user identities). To achieve this, fetch the user's profile as shown in the [User Sessions](/quickstart/native/ios-swift/03-user-sessions#validate-an-accesstoken) tutorial:
```

Comment:
This is redundant for OIDC flow (default now), can get rid of this as it's covered in calling the mgmt API

Reply:
Just to make sure - the whole "Retrieve the linked accounts" section or just the paragraph in the line 58?

Reply:
Remove this text